rh-summit-2018/basics.adoc

```diff
@@ -6,7 +6,7 @@ It consists of:
 * an OpenShift cluster with one master node and three worker nodes;
 * a workstation;
 
-During the lab you will interact with the OpenShift cluster via CLI from the workstation and with the OpenShift from the browser available on your station.
+During the lab you will interact with the OpenShift cluster via CLI from the workstation and from the browser available on your station.
```

Comment:
I think the missing words were 'web console' :)

So it should rather be `with the OpenShift web console`.

Reply:
yeah thanks it sounds better :-)